resources/themes/pterodactyl/admin/nodes/new.blade.php

```diff
@@ -32,15 +32,16 @@
                         <input type="text" name="name" id="pName" class="form-control" value="{{ old('name') }}"/>
                         <p class="text-muted small">Character limits: <code>a-zA-Z0-9_.-</code> and <code>[Space]</code> (min 1, max 100 characters).</p>
                     </div>
+                    <div class="form-group">
+                        <label for="pDescription" class="form-label">Description</label>
+                        <textarea name="description" id="pDescription" rows="4" class="form-control" style="resize: vertical;">{{ old('description') }}</textarea>
```

Comment:
Please remove the `resize:vertical` to match the way the rest of the panel works.